packages/terra-button-group/src/ButtonGroupButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { PropTypes } from 'react';
```

Comment:
Are you suggesting that `<ButtonGroup.Button />` simply wrap a `<Button />` and offer a scoped down api? Assuming the api of button group is a subset I'd be fine with that implementation.(@jmsv6d, could you verify?) Though i don't think we have to go so far as nulling custom props if people send them in. At that point you're going beyond the supported api at your own risk. Secondly would you mind if we pushed that change in as a separate pull request? It should be a passive api change and I'd like to break the changes up into smaller pull requests.